docs/02-app/02-api-reference/08-next-cli.mdx

```diff
@@ -146,7 +146,7 @@ To learn more about Telemetry, [please read this document](https://nextjs.org/te
 ## Next Info
 
 `next info` prints relevant details about the current system which can be used to report Next.js bugs.
-This information includes Operating System platform/arch/version, Binaries (Node.js, npm, Yarn, pnpm) and npm package versions (`next`, `react`, `react-dom`).
+This information includes Operating System platform/arch/version, Binaries (Node.js, npm, Yarn, pnpm, Bun) and npm package versions (`next`, `react`, `react-dom`).
```

Comment:
Your PR didn't change the behavior of `next info` so this isn't true

Reply:
Apologies, fixed